nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/GetHDFSFileInfo.java

```diff
@@ -264,8 +263,6 @@ protected void init(final ProcessorInitializationContext context) {
     @Override
     public void onPropertyModified(PropertyDescriptor descriptor, String oldValue, String newValue) {
```

Comment:
This method can be deleted now because it does not do anything just calls the super.